src/webui/www/private/scripts/client.js

```diff
@@ -967,7 +967,7 @@ window.addEvent('load', function() {
 
 function registerMagnetHandler() {
     if (typeof navigator.registerProtocolHandler !== 'function') {
```

Comment:
I think we can give a bit better message by detecting whether it is using HTTPS. Such as:
```js
if (typeof navigator.registerProtocolHandler !== 'function') {
    if (window.location.protocol !== 'https:')
        alert(QBT_TR("...")QBT_TR[CONTEXT=MainWindow]);  // wrapped in `QBT_TR()` to allow i18n/translation
    else
        alert(QBT_TR("Your browser does not support this feature")QBT_TR[CONTEXT=MainWindow]);  // wrapped in `QBT_TR()` to allow i18n/translation
}
```

Reply:
What if the browser just doesn't support `navigator.registerProtocolHandler()`? Then the HTTPS message would give false hope.

Reply:
>What if the browser just doesn't support navigator.registerProtocolHandler()? 

To users that can't be really known without trying HTTPS first, isn't?
I think it is ideal this way, but we can wait for others opinion.